test/suites/container_devices_nic_bridged.sh

```diff
@@ -648,6 +648,18 @@ test_container_devices_nic_bridged() {
   grep -F "192.0.2.232" "${LXD_DIR}/networks/${brName}/dnsmasq.hosts/foo.eth0"
   lxc delete -f foo
 
+  # Test container without extra network configuration can be restored from backup.
+  lxc init testimage foo -p "${ctName}"
+  lxc export foo foo.tar.gz
+  lxc import foo.tar.gz foo2
```

Comment:
lets delete foo.tar.gz here once its imported so we lessen the chance of leaving it behind.

Reply:
effectively reducing the lifetime of foo.tar.gz